engine/src/main/java/org/terasology/rendering/shader/ShaderParametersSSAO.java

```diff
@@ -92,17 +90,19 @@ public void initialParameters(Material material) {
     public void applyParameters(Material program) {
         super.applyParameters(program);
 
-        FBO scene = CoreRegistry.get(DisplayResolutionDependentFBOs.class).get(DefaultDynamicFBOs.READ_ONLY_GBUFFER.getName()); // TODO: switch from CoreRegistry to Context.
+        DisplayResolutionDependentFBOs displayResolutionDependentFBOs = CoreRegistry.get(DisplayResolutionDependentFBOs.class); // TODO: switch from CoreRegistry to Context.
+
+        FBO sceneOpaqueFbo = displayResolutionDependentFBOs.get(new ResourceUrn("engine:sceneOpaque"));
```

Comment:
@emanuele3d You left out an instance xD Lemme do it for you :D
"ResourcesUrn into a static final field please."